mmdet/evaluation/metrics/crowdhuman_metric.py

```diff
@@ -38,9 +38,8 @@ class CrowdHumanMetric(BaseMetric):
         outfile_prefix (str, optional): The prefix of json files. It includes
             the file path and the prefix of filename, e.g., "a/b/prefix".
             If not specified, a temp file will be created. Defaults to None.
-        file_client_args (dict): Arguments to instantiate a FileClient.
-            See :class:`mmengine.fileio.FileClient` for details.
-            Defaults to ``dict(backend='disk')``.
+        backend_args (dict, optional): Arguments to instantiate the
+            corresponding backend. Defaults to None.
```

Comment:
1. update docstrings to indicate that this argument is added since MMDet>=3.0.0rc7, before MMDet 3.0.0rc6 it is file_client_args